intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/inspection/Link.kt

```diff
@@ -0,0 +1,79 @@
+@file:OptIn(ApolloInternal::class)
+
+package com.apollographql.ijplugin.inspection
+
+import com.apollographql.apollo3.annotations.ApolloInternal
+import com.apollographql.apollo3.ast.GQLDirectiveDefinition
+import com.apollographql.apollo3.ast.NULLABILITY_VERSION
+import com.apollographql.apollo3.ast.nullabilityDefinitions
+import com.apollographql.ijplugin.util.quoted
+import com.apollographql.ijplugin.util.unquoted
+import com.intellij.lang.jsgraphql.psi.GraphQLArrayValue
+import com.intellij.lang.jsgraphql.psi.GraphQLDirective
+import com.intellij.lang.jsgraphql.psi.GraphQLElement
+import com.intellij.lang.jsgraphql.psi.GraphQLFile
+import com.intellij.lang.jsgraphql.psi.GraphQLNamedElement
+import com.intellij.lang.jsgraphql.psi.GraphQLSchemaDefinition
+import com.intellij.lang.jsgraphql.psi.GraphQLSchemaExtension
+import com.intellij.lang.jsgraphql.psi.GraphQLStringValue
+
+const val NULLABILITY_URL = "https://specs.apollo.dev/nullability/$NULLABILITY_VERSION"
+
+val NULLABILITY_DIRECTIVES: Map<String, Collection<String>> by lazy {
+  nullabilityDefinitions(NULLABILITY_VERSION)
+      .filterIsInstance<GQLDirectiveDefinition>()
+      .map { it.name to it.arguments.map { it.name } }
+      .toMap()
+}
+
+const val CATCH = "catch"
+const val CATCH_TO = "CatchTo"
```

Comment:
For the record, we just discussed this and it would be better to not hardcode any definition names in AKIJP. Instead, AKIJP can read from the `nullability` and `kotlin_labs` definitions.

* unresolved `nullability` imports are an error.
* unresolved `kotlin_labs` imports are an error too (TBC). 4.0 is lenient and imports them automatically but we should enforce this so might just be easier to have the same handling there.

Aliased imports (`{ name: "@cactch", to: `"mycatch" }`) cannot have a quick fix, which is expected, same as in Kotlin.

Reply:
`kotlin_labs` warning in latest commit:


Uploading Screen Recording 2023-12-22 at 11.15.07.mov…